net-mail/automx2/automx2-2022.0.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9,10} )
+
+inherit distutils-r1
+
+DESCRIPTION="Email client autoconfiguration service"
+HOMEPAGE="https://automx.org/"
+SRC_URI="https://github.com/rseichter/automx2/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+
+BDEPEND="acct-user/automx2"
```

Comment:
It doesn't look like `fowners` or `chown` is called in this ebuild, so this should be in RDEPEND I reckon.

Reply:
I thought that acct-user/* was always used during build time, but if you wish to move it to RDEPEND, I have no objections.